SPEC.md

```diff
@@ -51,401 +25,700 @@ Released versions of the spec are available as Git tags.
 ## Overview
 
 This document proposes a generic plugin-based networking solution for application containers on Linux, the _Container Networking Interface_, or _CNI_.
-It is derived from the rkt Networking Proposal, which aimed to satisfy many of the design considerations for networking in [rkt][rkt-github].
 
-For the purposes of this proposal, we define two terms very specifically:
-- _container_ can be considered synonymous with a [Linux _network namespace_][namespaces]. What unit this corresponds to depends on a particular container runtime implementation: for example, in implementations of the [App Container Spec][appc-github] like rkt, each _pod_ runs in a unique network namespace. In [Docker][docker], on the other hand, network namespaces generally exist for each separate Docker container.
-- _network_ refers to a group of entities that are uniquely addressable that can communicate amongst each other. This could be either an individual container (as specified above), a machine, or some other network device (e.g. a router). Containers can be conceptually _added to_ or _removed from_ one or more networks.
+For the purposes of this proposal, we define three terms very specifically:
+- _container_ can be considered synonymous with a [Linux _network namespace_][namespaces], though the actual isolation technology is not defined by the specification. What unit this corresponds to depends on a particular container runtime implementation.
```

Comment:
I think we should be explicit that non-containers such as VMs are allowed, and the fields are just re-purposed